Bynder/Sdk/Service/Upload/FileUploader.cs

```diff
@@ -302,9 +337,18 @@ private async Task<FinalizeResponse> FinalizeUploadAsync(UploadRequest uploadReq
                 S3Filename = $"{uploadRequest.S3Filename}/p{chunkNumber}",
                 Chunks = chunkNumber.ToString()
             };
+            var requestParameters = "";
+            if (customParameters != null)
```

Comment:
@quirijnslings Wanted to double check what the custom params look like for a finalize upload request, do you have an example request/endpoint with the custom params added?

Reply:
The only relevant custom parameter I know of is async_conversion=true. This tells the API to save the asset and return directly, instead of waiting for all derivatives etc to be generated. If you upload a large tif you will see that it becomes much faster with this param.
The reason I did it in such a roundabout way, rather than simply adding a Boolean 'AsynConversion' property to the query, is that this parameter is not documented. It is sometimes used by Bynder consultants to boost performance in large scale upload scenarios